packages/salesforcedx-vscode-apex/src/views/icons/iconHelpers.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { extensionUris } from '@salesforce/salesforcedx-utils-vscode';
+import { VSCODE_APEX_EXTENSION_NAME } from '../../constants';
+import { iconKey, ICONS } from './IconsEnum';
+
+/**
+ * Get the Uri for an icon located in the resources directory.
+ * @param key A key from the {@link IconsEnum}
+ * @returns The Uri to the icon image.
+ */
```

Comment:
![Screenshot 2023-02-16 at 1 02 26 AM](https://user-images.githubusercontent.com/76090802/219292052-b53354d7-57f9-40aa-a375-257a60b770a2.png)

Note the param here is limited to the key values from the Icons const (which are also the keys for the IconsEnum). 
This allows us to use this method safely always limiting the param to the available icons (or getting a compile error)

Reply:
Sweeet